app/main/helpers/search_helpers.py

```diff
@@ -103,6 +103,7 @@ def clean_request_args(request_args, lot_filters, lots_by_slug, for_aggregation=
 
     if not for_aggregation:
         restore_keys.append('page')
+        restore_keys.append('parentCategory')
```

Comment:
Below this we validate whether the `lot` is a valid lot before restoring it. Should we be doing this for `parentCategory` as well? (How we'd do that, I have no idea 😹)

Reply:
(Also I'm sure you're looking at tests already, but we should definitely also update `test_clean_request_args` with `parentCategory`)